pkg/sql/sequence.go

```diff
@@ -432,6 +432,9 @@ func resolveColumnItemToDescriptors(
 	var tableName tree.TableName
 	if columnItem.TableName != nil {
```

Comment:
Super nit: does it make more sense to do 
```
if columnItem.tableName == nil {
  return error...
}

tableName = columnItem.TableName.ToTableName()
```

Reply:
yes for sure! thanks for the catch